recipes/default.rb

```diff
@@ -14,10 +14,18 @@
 # limitations under the License.
 #
 
+ruby_block 'ensure node can resolve API FQDN' do
+  extend ChefAnalyticsCookbook::Helpers
+  block { repair_api_fqdn }
+  only_if { api_fqdn_available? }
+  not_if { api_fqdn_resolves? }
+end
```

Comment:
As noted in @binamov PR, this was pulled in from https://github.com/chef-cookbooks/chef-server/blob/master/libraries/helpers.rb.

There were notes in that the FQDN is rather important to Analytics, so I thought it prudent to include it in this PR to make the cookbook more resilient, however indeed I do prefer to make it as explicit as possible we need this set. That block is intended to ensure we're setting it or applying the CYA policy.